WORKSPACE

```diff
@@ -846,146 +854,161 @@ http_archive(
 )
 
-new_http_archive(
+http_archive(
     name = "com_github_cereal",
     urls = ["https://github.com/USCiLab/cereal/archive/v1.2.2.tar.gz"],
     strip_prefix = "cereal-1.2.2",
-    build_file = "third_party/cereal/cereal.BUILD",
+    build_file = "@//:third_party/cereal/cereal.BUILD",
     sha256 = "1921f26d2e1daf9132da3c432e2fd02093ecaedf846e65d7679ddf868c7289c4",
 )
 
-new_http_archive(
+http_archive(
     name = "com_github_jbeder_yaml_cpp",
     urls = ["https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-0.6.2.tar.gz"],
     strip_prefix = "yaml-cpp-yaml-cpp-0.6.2",
-    build_file = "third_party/yaml-cpp/yaml.BUILD",
+    build_file = "@//:third_party/yaml-cpp/yaml.BUILD",
+    sha256 = "e4d8560e163c3d875fd5d9e5542b5fd5bec810febdcba61481fe5fc4e6b1fd05",
 )
 
-new_http_archive(
+http_archive(
     name = "com_github_danmar_cppcheck",
     urls = ["https://github.com/danmar/cppcheck/archive/1.87.zip"],
     strip_prefix = "cppcheck-1.87",
-    build_file = "third_party/cppcheck/cppcheck.BUILD",
+    build_file = "@//:third_party/cppcheck/cppcheck.BUILD",
+    sha256 = "b3de7fbdc1a23d7341b55f7f88877e106a76847bd5a07fa721c07310b625318b",
 )
 # end 3rdparty C++ dependencies
 
 # for helm
-new_http_archive(
+http_archive(
     name = "helm_mac",
-    url = "https://storage.googleapis.com/kubernetes-helm/helm-v2.7.2-darwin-amd64.tar.gz",
+    urls = ["https://storage.googleapis.com/kubernetes-helm/helm-v2.7.2-darwin-amd64.tar.gz"],
     strip_prefix = "darwin-amd64",
-    build_file = "third_party/helm/helm.BUILD",
+    build_file = "@//:third_party/helm/helm.BUILD",
     sha256 = "5058142bcd6e16b7e01695a8f258d27ae0b6469caf227ddf6aa2181405e6aa8e",
 )
 
-new_http_archive(
+http_archive(
     name = "helm_linux",
-    url = "https://storage.googleapis.com/kubernetes-helm/helm-v2.7.2-linux-amd64.tar.gz",
+    urls = ["https://storage.googleapis.com/kubernetes-helm/helm-v2.7.2-linux-amd64.tar.gz"],
     strip_prefix = "linux-amd64",
-    build_file = "third_party/helm/helm.BUILD",
+    build_file = "@//:third_party/helm/helm.BUILD",
     sha256 = "9f04c4824fc751d6c932ae5b93f7336eae06e78315352aa80241066aa1d66c49",
 )
 # end helm
 
 # for docker image building
 http_archive(
     name = "io_bazel_rules_docker",
-    urls = ["https://github.com/bazelbuild/rules_docker/archive/v0.4.0.tar.gz"],
-    strip_prefix = "rules_docker-0.4.0",
-    sha256 = "6dede2c65ce86289969b907f343a1382d33c14fbce5e30dd17bb59bb55bb6593",
+    urls = ["https://github.com/bazelbuild/rules_docker/archive/v0.7.0.tar.gz"],
+    strip_prefix = "rules_docker-0.7.0",
+    sha256 = "aed1c249d4ec8f703edddf35cbe9dfaca0b5f5ea6e4cd9e83e99f3b0d1136c3d",
 )
 
 load(
-    "@io_bazel_rules_docker//container:container.bzl",
-    "container_pull",
+    "@io_bazel_rules_docker//repositories:repositories.bzl",
     container_repositories = "repositories",
 )
 
-# This is NOT needed when going through the language lang_image
-# "repositories" function(s).
 container_repositories()
 
+load(
```

Comment:
Is this required by new bazel? If not, maybe better to be in a separated PR I think.

Reply:
The docker rules need to be updated for compatibility with Bazel, this statement simply loads the `container_pull` rule from its new location.

Reply:
Got it. Thanks.